test/integration_test.js

```diff
@@ -1018,76 +1017,55 @@ describe("Integration tests", () => {
-        constructor(char) {
-          super();
-          this.char = char;
-        }
-        encode(record) {
-          return Object.assign({}, record, {title: record.title + this.char});
-        }
-        decode(record) {
-          return Object.assign({}, record, {title: record.title.slice(0, -1)});
-        }
+      function createTransformer(char) {
+        return {
+          encode(record) {
+            return Object.assign({}, record, {title: record.title + char});
+          },
+          decode(record) {
+            return Object.assign({}, record, {title: record.title.slice(0, -1)});
+          }
+        };
       }
 
-      const ES5TitleCharTransformer = Kinto.createRemoteTransformer({
-        constructor: function(char) {
-          this.char = char;
-        },
-        encode: function(record) {
-          return Object.assign({}, record, {title: record.title + this.char});
-        },
-        decode: function(record) {
-          return Object.assign({}, record, {title: record.title.slice(0, -1)});
-        }
-      });
-
-      function testTransformer(name, Transformer) {
-        describe(name, () => {
-          beforeEach(() => {
-            tasks = kinto.collection("tasks", {
-              remoteTransformers: [
-                new Transformer("!"),
-                new Transformer("?")
-              ]
-            });
-
-            return Promise.all([
-              tasks.create({id: uuid4(), title: "abc"}, {useRecordId: true}),
-              tasks.create({id: uuid4(), title: "def"}, {useRecordId: true}),
-            ]);
-          });
+      beforeEach(() => {
+        tasks = kinto.collection("tasks", {
+          remoteTransformers: [
+            createTransformer("!"),
+            createTransformer("?")
+          ]
+        });
 
-          it("should list published records unencoded", () => {
-            return tasks.sync()
-              .then(res => res.published.map(x => x.title).sort())
-              .should.become(["abc", "def"]);
-          });
+        return Promise.all([
+          tasks.create({id: uuid4(), title: "abc"}, {useRecordId: true}),
+          tasks.create({id: uuid4(), title: "def"}, {useRecordId: true}),
+        ]);
+      });
 
-          it("should store encoded data remotely", () => {
-            return tasks.sync()
-              .then(_ => {
-                return fetch(`${TEST_KINTO_SERVER}/buckets/default/collections/tasks/records`, {
-                  headers: {"Authorization": "Basic " + btoa("user:pass")}
-                });
-              })
-              .then(res => res.json())
-              .then(res => res.data.map(x => x.title).sort())
-              .should.become(["abc!?", "def!?"]);
-          });
+      it("should list published records unencoded", () => {
+        return tasks.sync()
+          .then(res => res.published.map(x => x.title).sort())
+          .should.become(["abc", "def"]);
+      });
```

Comment:
Ah no, it's just test refactoring now we don't need two test suites for es5/6 transformers :)